### PR TITLE
Konvoy release docs

### PR DESCRIPTION
**Pull request by dkoshkin**
* konvoy: Update 1.8 docs

### CODE REVIEW

File: pages/dkp/konvoy/1.8/install/install-aws/advanced-provisioning/index.md [hunk 1]
```diff
@@ -409,7 +409,168 @@ kubernetes.io/cluster = __CLUSTER_NAME__
...
+      rootVolumeSize: 10
+      rootVolumeType: gp2
+      type: m5.large
+      aws:
+        securityGroupIDs:
+          - sg-0e2300d267ff60ce8
+...
+```
+
+*NOTE* it is not necessary to use the same security group list, each section
```
Comment (russurquhart1) [inline pages/dkp/konvoy/1.8/install/install-aws/advanced-provisioning/index.md:479]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>It is not necessary to use the same security group list, each section could have a different list.</p>

```

File: pages/dkp/konvoy/1.8/install/install-aws/advanced-provisioning/index.md [hunk 2]
```diff
@@ -409,7 +409,168 @@ kubernetes.io/cluster = __CLUSTER_NAME__
...
+could have a different list.
```
Comment (russurquhart1) [inline pages/dkp/konvoy/1.8/install/install-aws/advanced-provisioning/index.md:480]:
```suggestion
```

File: pages/dkp/konvoy/1.8/install/install-aws/index.md [hunk 3]
```diff
@@ -141,6 +147,45 @@ Below is the minimal IAM policy required:
 }
 ```
 
+If you wish to use an existing VPC, the following permissions are not necessary and can be removed from the set above.
```
Comment (russurquhart1) [inline pages/dkp/konvoy/1.8/install/install-aws/index.md:150]:
```suggestion
If you want to use an existing VPC, the following permissions are not necessary and can be removed from the set above.
```

File: pages/dkp/konvoy/1.8/install/install-onprem/index.md [hunk 4]
```diff
@@ -527,6 +522,19 @@ Note that if your stateful workload is using a [local persistent volume][local_p
 If the node fails, the stateful workload might lose its data.
 If you cannot tolerate this limitation, you should consider other storage options.
 
+# Configure NTP on nodes
+
+By default, Konvoy will install and configure `chrony` as an NTP client used to synchronize time on the control-plane and worker machines.
```
Comment (russurquhart1) [inline pages/dkp/konvoy/1.8/install/install-onprem/index.md:527]:
```suggestion
By default, Konvoy instalsl and configures `chrony` as an NTP client used to synchronize time on the control-plane and worker machines.
```

**Review by russurquhart1 — APPROVED** [review]

File: pages/dkp/konvoy/1.7/install/install-onprem/index.md [hunk 5]
```diff
@@ -18,7 +18,9 @@ Before installing, verify that your environment meets the following basic requir
 
 * [Docker][install_docker] version 18.09.2 or later. You must have Docker installed on the host where the Konvoy command line interface (CLI) will run. For example, if you are installing Konvoy on your laptop computer, be sure the laptop has a supported version of Docker.
 
-* [kubectl][install_kubectl] v1.19.8 or later. You must have `kubectl` installed on the host, where the Konvoy command line interface (CLI) runs, to enable interaction with the running cluster.
+* [kubectl][install_kubectl] v1.19.9 or later. You must have `kubectl` installed on the host, where the Konvoy command line interface (CLI) runs, to enable interaction with the running cluster.
+
+* Konvoy cannot be run from a host that belongs to a Kubernetes cluster, you must have a separate host.
```
Comment (cneth) [inline pages/dkp/konvoy/1.7/install/install-onprem/index.md:23]: This is redundant with the next line in the file.
```suggestion
```

File: pages/dkp/konvoy/1.8/install/http-proxy/index.md [hunk 6]
```diff
@@ -100,7 +100,9 @@ spec:
       noProxy: [ "172.0.0.0/16", ".intra.net" ]
 ```
 
-In a cluster with the default configuration, the `kommander` addon is installed. This addon needs access to the Internet to list the available `docker.io/mesosphere/konvoy` Docker images. To configure the addon to use the proxy settings, modify its `values` field as follows:
+In a cluster with the default configuration, the `kommander` and `gatekeeper` addons are installed. Gatekeeper will automatically configure the right proxy settings to `kommander`. If on the other hand, `gatekeeper` is not enabled then you will have to manually configure `kommander`.
```
Comment (cneth) [inline pages/dkp/konvoy/1.8/install/http-proxy/index.md:103]:
```suggestion
In a cluster with the default configuration, the `kommander` and `gatekeeper` addons are installed. Gatekeeper will automatically configure the right proxy settings for `kommander`. If `gatekeeper` is not enabled then you will have to manually configure the proxy settings for `kommander`.
```

File: pages/dkp/konvoy/1.8/install/install-onprem/index.md [hunk 7]
```diff
@@ -527,6 +522,19 @@ Note that if your stateful workload is using a [local persistent volume][local_p
...
+You can disable this by configuring your cluster as follows:
```
Comment (cneth) [inline pages/dkp/konvoy/1.8/install/install-onprem/index.md:528]: Let's make it clear you should only disable this if you already have something else doing this:

```suggestion
If your machines already have time synchronization software installed and configured, you can disable the installation of `chrony` by configuring your cluster as follows:
```

**Review by cneth — CHANGES_REQUESTED** [review]

**Review by cneth — APPROVED** [review]

**Review by gracedo — APPROVED** [review]

**Pull request merged**